packages/terra-status-view/src/StatusView.jsx

```diff
@@ -0,0 +1,254 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Divider from 'terra-divider';
+import styles from './StatusView.scss';
+
+const cx = classNames.bind(styles);
+
+const variants = {
+  NODATA: 'no-data',
+  NOMATCHINGRESULTS: 'no-matching-results',
+  NOTAUTHORIZED: 'not-authorized',
+  ERROR: 'error',
+  NOSERVICE: 'no-service',
+  CUSTOM: 'custom',
+};
+
+const propTypes = {
+  /**
+   * An array of actionable buttons to display.
+   */
+  children: PropTypes.node,
+
+  /**
+   *  Determines if the content should be aligned vertically at the top of the container rather than in the middle.
+   */
+  isAlignedTop: PropTypes.bool,
+
+  /**
+   * Determines if the glyph should be displayed.
+   */
+  isGlyphHidden: PropTypes.bool,
+
+  /**
+   * The descriptive text to display.
+   */
+  message: PropTypes.string,
+
+  /**
+   * The required status title to display. Status views with variants of type no-data, no-matching-results,
+   * not-authorized, and error will have defaulted values unless overridden with this prop.
+   */
+  title: PropTypes.string,
+
+  /**
+   * Determines the type of icon to display.
+   */
+  variant: PropTypes.oneOf([
+    variants.NODATA,
+    variants.NOMATCHINGRESULTS,
+    variants.NOTAUTHORIZED,
+    variants.ERROR,
+    variants.NOSERVICE,
+    variants.CUSTOM,
+  ]),
+};
+
+const defaultProps = {
+  children: [],
+  isAlignedTop: false,
+  isGlyphHidden: false,
+  message: undefined,
+  title: undefined,
+  variant: variants.ERROR,
+};
+
+const contextTypes = {
```

Comment:
why is this needed?

Reply:
This could use the `injectIntl` HOC factory instead: https://github.com/yahoo/react-intl/wiki/API#injection-api

Reply:
it was added for internationalization of the default titles.  is what has been done in other components such as DemographicsBanner, Field, Select, Alert, DatePicker